merged-data.csv

```diff
@@ -0,0 +1,1121 @@
+station_name,datetime,pm_10_conc,pm_10_AQI,pm_10_cat,pm_25_conc,pm_25_AQI,pm_25_AQI_cat
```

Comment:
maybe we should have a cleanup process to delete the csv files after they're loaded?